bootstrap/build.cmd

```diff
@@ -0,0 +1,21 @@
+@ECHO off
+SETLOCAL
+
+SET KOREBUILD_REPO=https://github.com/aspnet/KoreBuild
+SET KOREBUILD_BRANCH=victorhu/init
+
+SET SCRIPT_FOLDER=%~dp0
+SET KOREBUILD_SCRIPT=%SCRIPT_FOLDER%\KoreBuild.cmd
+
+if EXIST "%KOREBUILD_SCRIPT%" GOTO endbootstrap
+
+SET TEMP_CLONE_LOCATION=%TEMP%\KoreBuildClone
+ECHO Downloading KoreBuild in %SCRIPT_FOLDER%...
+
+if EXIST %TEMP_CLONE_LOCATION% RMDIR %TEMP_CLONE_LOCATION% /S /Q
+git clone --branch %KOREBUILD_BRANCH% --depth 1 %KOREBUILD_REPO% %TEMP_CLONE_LOCATION%
```

Comment:
Not every windows machine has git in their path. It seems we've replaced nuget.exe with git.exe.